src/documents/ComponentConfigurationOptions.md

```diff
@@ -179,80 +195,51 @@ The following options are supported by all component grades:
   </tr>
 </table>
 
-### dynamicComponents ###
+### `distributeOptions` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object containing named definitions of the component's dynamic subcomponents</td>
-  </tr>
-  <tr>
-    <th>Notes</th>
-    <td>Some special context names may be available within the subcomponent's definition block, for example <code>{source}</code> and <code>{sourcePath}</code> or <code>{arguments}</code>. This framework facility will be replaced by a more declarative equivalent in time and should be used with caution.</td>
+    <td>A record directing the framework to distribute options from this component to one or more other components in the component tree. Either a single, an <code>Array</code> or <code>Object</code> holding these records is supported.
```

Comment:
"Either a single, an Array or Object holding these records..." is incomplete (a single what? an array of what? Please clarify.